WordPress/Resources/ar.lproj/Localizable.strings

```diff
@@ -9851,6 +9854,9 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 /* This text is required by Apple to be part of the translations but is not shown to the users. You can use the English version verbatim without translation for this entry. */
 "ios-widget.tVvJ9c" = "تحديد هدف الموقع";
 
+/* Title of the Jetpack powered badge. */
+"jetpack.branding.badge_banner.title" = "مدعوم بواسطة Jetpack";
```

Comment:
Maybe we should add a Peril check that all new localized strings use the reverse-DNS syntax? This is the only one that does in the batch that came through with this round of downloads.

I'm thinking of Peril because a SwiftLint rule would pick up _all_ existing localizations, and that would be quite noisy. 

The downside of Peril is that it's ignorable and also gives delayed feedback, you need to open the PR to know about it when it would be better to get told about it locally by Xcode. Still, better that then no feedback and the pattern not being adopted? 🤔

Reply:
💯  This is something we already suggested in the past, so I'm all for it.